frontend/public/tokener.html

```diff
@@ -2,33 +2,35 @@
 <html lang="en">
   <head>
     <script type="text/javascript">
-      // eslint-disable-next-line no-var
-      var json = [[.]];
-      var error;
-      try {
-        ['exp', 'email', 'name', 'userID'].forEach(function (key) {
-          localStorage.setItem(key, json[key]);
-        });
-      } catch (e) {
-        error = e;
-        console.error(e);
-        document.body.append('localStorage must be enabled:');
-        document.body.append(e.message || e.toString());
-      }
-      if (!error) {
-        var next = localStorage.getItem('next') || '';
-        if (next) {
-          if (next[0] === '/' && json.loginSuccessURL.substr(-1) === '/') {
-            while (next[0] === '/') {
-              next = next.substr(1); //remove any slash in front of "next"
+      window.onload = function() {
+        // eslint-disable-next-line no-var
+        var json = [[.]];
+        var error;
+        try {
+          ['exp', 'email', 'name', 'userID'].forEach(function (key) {
+            localStorage.setItem(key, json[key]);
+          });
+        } catch (e) {
+          error = e;
+          console.error(e);
+          document.body.append('localStorage must be enabled:');
+          document.body.append(e.message || e.toString());
+        }
+        if (!error) {
```

Comment:
Digging into the code, I don't believe the `localStorage` items are strictly required to continue, particularly for OpenShift where we user information is not stored there. We track login state in a cookie, and user preferences have moved to a config map. We might just continue and not fail on an error.

Reply:
@spadgett so do we want to keep the `onload()` method? Or just check if the `document.body` is not `null` when trying to write into it ?